core/templates/components/question-directives/question-misconception-selector/question-misconception-selector.component.html

```diff
@@ -0,0 +1,38 @@
+Selected Misconception: <[$ctrl.selectedMisconception && $ctrl.selectedMisconception.getName()]>
+<div ng-repeat="(skillId, misconceptions) in $ctrl.misconceptionsBySkill">
+  <div ng-repeat="misconception in misconceptions">
+    <button class="btn btn-secondary misconception-button" style="width: 100%" ng-click="$ctrl.selectMisconception(misconception, skillId)">
+      <b style="display: inline;"><[misconception.isMandatory() ? '' : ' (Optional) ']><[misconception.getName()]></b>
+      <angular-html-bind class="misconception-notes-container" html-data="misconception.getNotes()">
+      </angular-html-bind>
+    </button>
+    <div ng-if="$ctrl.selectedMisconceptionSkillId === skillId && $ctrl.selectedMisconception.getId() === misconception.getId()">
+      <label class="misconception-label"> Note to creators: </label>
+      <angular-html-bind html-data="misconception.getNotes()">
+      </angular-html-bind>
+      <label class="misconception-label"> Misconception Feedback: </label>
+      <angular-html-bind html-data="misconception.getFeedback()">
+      </angular-html-bind>
+    </div>
+  </div>
+</div>
+<span ng-click="$ctrl.toggleMisconceptionFeedbackUsage()" aria-hidden="true">
+  <i ng-if="!$ctrl.misconceptionFeedbackIsUsed" class="material-icons md-18 misconception-feedback-usage-checkbox">&#xE835;</i>
+  <i ng-if="$ctrl.misconceptionFeedbackIsUsed" class="material-icons md-18 misconception-feedback-usage-checkbox">&#xE834;</i>
+  Use misconception feedback as answer group feedback.
+</span>
+<!-- <button class="btn btn-success" ng-disabled="$ctrl.selectedMisconception === null" ng-click="save()"> Use Misconception </button> -->
```

Comment:
Please do not commit commented-out code. Proofread the PR before sending for review.